lib/modules/manager/terraform/base.ts

```diff
@@ -0,0 +1,65 @@
+import is from '@sindresorhus/is';
+import { regEx } from '../../../util/regex';
+import { TerraformProviderDatasource } from '../../datasource/terraform-provider';
+import type { PackageDependency } from '../types';
+import type { ProviderLock } from './lockfile/types';
+import { getLockedVersion, massageProviderLookupName } from './util';
+
+export abstract class DependencyExtractor {
+  /**
+   * Get a list of signals which can be used to scan for potential processable content
+   * @return a list of content signals
+   */
+  abstract getCheckList(): string[];
+
+  /**
+   * Extract dependencies from a HLC object
+   * @param hclRoot HLC parsing artifact.
```

Comment:
```suggestion
   * Extract dependencies from a HCL object
   * @param hclRoot HCL parsing artifact.
```

I think you meant `HCL` here?

Reply:
Ah, yes that's correct. Thanks.